products/qt.md

```diff
@@ -13,9 +13,7 @@ auto:
 releases:
 -   releaseCycle: "6.4"
     eol: 2023-03-30
-    latest: "6.4.1"
-    link: https://www.qt.io/blog/qt-6.4-released
-    # remove link tag after we got a point versions
+    latest: "6.4.0"
```

Comment:
Qt version has been downgraded in order to be able to test the changes.

Reply:
btw thank you for Qt's new version update